src/lib/collab/provider.js

```diff
@@ -7,6 +7,24 @@ import get from 'lodash/get'
 import debounce from 'lodash/debounce'
 
 const jsonTransformer = new JSONTransformer()
+// Using the jsonTransformer directly introduce a bug for empty documents
+// with only one paragraph. The transformed document has no content
+// (not even the paragraph) and makes the Editor to always when think
+// it is as version=0. This would break the catchup logic.
+// This fix path this unique case, creating a correct empty document with
```

Comment:
```suggestion
// This fix path this unique case, creating a correct empty document with
```

this patch fix this unique case?